status/status.go

```diff
@@ -371,3 +371,11 @@ func GenerateSummary(state intgteststat.IntegrationTestStatus, snapshotName, sce
 
 	return summary, nil
 }
+
+func getConsoleName() string {
+	consoleName := os.Getenv("CONSOLE_NAME")
+	if consoleName == "" {
+		return "CONSOLE_NAME_NOT_AVAILABLE"
```

Comment:
```suggestion
		return "Integration Service"
```
I think we should return upstream name in this case, if somebody runs it as a standalone service. If you run as part of the product, then you can set CONSOLE_NAME

Reply:
Good idea!